docs/changelog.txt

```diff
@@ -316,6 +316,9 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Structures
 - New globals: ``soul_next_id``
 
+## Lua
+- Added ``profiler`` module to measure lua performance
+
```

Comment:
Wrong section - this is under 0.44.05-r1.

It's possible that you based this on an old commit. If that's the case, it should be manageable enough to fix on our end.